src/main/java/org/example/bookshop/model/Book.java

```diff
@@ -0,0 +1,43 @@
+package org.example.bookshop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
```

Comment:
Don't use @Data on JPA entities, it contains problematic annotations like ToString and EqualsAndHashcode
https://medium.com/@devchaghtai/understanding-and-resolving-lombok-data-pitfalls-in-jpa-hibernate-entities-ebc76000da18
Use @Getter @ Setter